sdk/containerorchestratorruntime/Azure.ResourceManager.ContainerOrchestratorRuntime/tests/Scenario/LoadBalancerTests.cs

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using Azure.Core;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.ContainerOrchestratorRuntime.Models;
+using Azure.ResourceManager.Kubernetes;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.ContainerOrchestratorRuntime.Tests.Tests
+{
+    [TestFixture]
+    public class LoadBalancerTests : ContainerOrchestratorRuntimeManagementTestBase
+    {
+        public LoadBalancerTests() : base(false)
+        {
+        }
+
+        [TestCase]
+        [RecordedTest]
+        public void CreateLoadBalancer()
+        {
+            var connectedCluster = ConnectedClusterResource.CreateResourceIdentifier("b9e38f20-7c9c-4497-a25d-1a0c5eef2108", "xinyuhe-canary", "test-cluster-euap-arc");
```

Comment:
Try to externalize those environment properties. You can put them in `ContainerOrchestratorRuntimeTestEnvironment`. See https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/core/Azure.Core.TestFramework#test-resource-creation-and-testenvironment

Reply:
Extracted the environment properties.